tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/CaeUserTests.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.UI;
+using Microsoft.Identity.Test.Integration.Infrastructure;
+using Microsoft.Identity.Test.LabInfrastructure;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Integration.NetFx.SeleniumTests
+{
+    [TestClass]
+    public class CaeUserTests
+    {
+        private readonly string _certificateName = "for-cca-testing";
+        private readonly string _confidentialClientID = "35dc5034-9b65-4a5d-ad81-73cca468c1e0"; //msidlab4.com app
+        private readonly string _tenantId = "72f988bf-86f1-41af-91ab-2d7cd011db47";
+        private readonly KeyVaultSecretsProvider _keyVaultProvider = new(KeyVaultInstance.MsalTeam);
+        private readonly HttpClient _httpClient = new();
+        private readonly TimeSpan _delayTimeout = TimeSpan.FromMinutes(5);
+        private readonly string[] _graphUserScope = ["User.Read"];
+        private static readonly TimeSpan s_loginTimeout = TimeSpan.FromMinutes(1);
+
+        public TestContext TestContext { get; set; }
+
+        /// <summary>
+        /// Login with the test user using Selenium; the cache should have a user token.
```

Comment:
nit: I prefer to add these comments in the test itself, e.g. `Debug.Write("1. Login with the...)`

Reply:
Oh, felt like having this as one top comment helped see the E2E flow clearer.